backend/src/main/java/de/roman/toolio/controller/InquiryPartController.java

```diff
@@ -23,9 +23,15 @@ public InquiryPartController(InquiryPartService inquiryPartService) {
         return inquiryPartService.listInquiryParts();
     }
 
+    @GetMapping("{id}")
+    public InquiryPart getInquiryById(@PathVariable String id){
+        return inquiryPartService.getInquiryById(id);
+    }
+
     @PostMapping
     public InquiryPart addInquiry(@RequestBody InquiryPart inquiryPartToBeAdded){
```

Comment:
i would define a separate model to make clear what the api user needs to define. When you add swagger you would see that you have to add an id but have no idea that it will be overwritten